frontend/appflowy_tauri/src/appflowy_app/components/layout/NavigationPanel/FolderItem.hooks.ts

```diff
@@ -36,6 +36,7 @@ export const useFolderEvents = (folder: IFolder, pages: IPage[]) => {
 
   useEffect(() => {
     void appObserver.subscribe({
+      /* these event is not firing */
       onAppChanged: (change) => {
```

Comment:
Is this callback still necessary? The update of page items can be realized through the next callback

Reply:
I've analyzed the code, so the next callback replaces the current one. I've deleted this callback, since it's never used.